contracts/components/Safes.sol

```diff
@@ -135,6 +159,7 @@ contract Safes {
             signalingPeriod: _signalingPeriod,
             endSignalTime: 0,
             latestSignalTime: 0,
+            dDay: block.timestamp + _DDay,
```

Comment:
Unlike `signalBased`, the DDay timestamp is an absolute time. _DDay itself should be a future timestamp.